extensions/json/build.gradle

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+//description 'JSON: Integrating Engine tables with JSON'
+
+dependencies {
+    api project(':Base')
+    api project(':Util')
+    api project(':engine-time')
+    api project(':table-api')
+    api project(':IO')
+    api project(':extensions-parquet-table')
+    api project(':extensions-kafka')
+    api project(':Configuration')
+    api project(':FishUtil')
+    api project(':TableLogger')
+
+
+    // Using io.confluent dependencies requires code in the toplevel build.gradle to add their maven repository.
+    // Note: the -ccs flavor is provided by confluent as their community edition. It is equivalent to the maven central
+    // version, but has a different version to make it easier to keep confluent dependencies aligned.
+    api 'org.apache.kafka:kafka-clients:7.1.1-ccs'
+    api 'io.confluent:kafka-avro-serializer:7.1.1'
```

Comment:
Think only because of `io.deephaven.kafka.ingest.JsonNodeUtil` — meant to move that here and make Kafka depend on this module, but didn't get around to it yet

Reply:
ok, moved that class & flipped the dependencies